v2/call_option.go

```diff
@@ -119,6 +121,38 @@ func (r *boRetryer) Retry(err error) (time.Duration, bool) {
 	return 0, false
 }
 
+// OnHTTPCodes returns a Retryer that retries if and only if
+// the previous attempt returns a googleapi.Error whose status code is stored in
+// cc. Pause times between retries are specified by bo.
+//
+// bo is only used for its parameters; each Retryer has its own copy.
+func OnHTTPCodes(cc []int, bo Backoff) Retryer {
```

Comment:
If this matches conventions of other APIs in this package please ignore, I did not check.

I wonder if this should be `func OnHTTPCodes(bo Backoff, codes ...int)`. Might be a little more ergonomic to work with.

Reply:
Yeah I copied this from OnCodes, but honestly, I like the variadic form better...I wouldn't say there is a convention in this package, so let's take the chance to use a better signature.